<launch>  
  <rosparam
    file="$(find usv16)/config/control_params.yaml"
    param="ctrl_params"
    command="load"
  />
  
  <param 
    name="robot_description" textfile='$(find usv16)/urdf/ship.urdf' 
  />
  <param 
    name="use_gui" value="True"
  /> 

  <node
    pkg="usv16"
    type="map_reference"
    name="map_reference"
  />
  <node 
    name="robot_state_publisher" 
    pkg="robot_state_publisher" 
    type="state_publisher"
  />
 <node
    pkg="rviz"
    type="rviz"
    name="rviz_node"
    args="-d $(find usv16)/config/rviz.rviz"
  />
</launch>
